### PR TITLE
api-derive documentation

### PR DESCRIPTION
**Pull request by Stefie**
Adresses #603

### CODE REVIEW

File: packages/types/README.md [hunk 1]
```diff
@@ -160,3 +159,12 @@ These types are not used in the runtime, but are rather used in RPC results:
 | [[RuntimeVersion]] | A [[Tuple]] that conatins the [[ApiId]] and [[U32]] version |
 | [[SignedBlock]] | A [[Block]] that has been signed and contains a [[Justification]] |
 | [[StorageChangeSet]] | A set of storage changes. It contains the [[Block]] hash and a list of the actual changes |
+
+## Derive types
+
+These types are are specific for the Polkadot-JS API, so you won't find a representation of them in the SCALE codec or the Substrate core. They are used in the [api-derive] methods:(https://www.npmjs.com/package/@polkadot/api-derive)
```
Comment (amaurymartiny) [inline packages/types/README.md]:
```suggestion
These types are are specific for the Polkadot-JS API, so you won't find a representation of them in the SCALE codec or the Substrate core. They are used in the [api-derive](https://www.npmjs.com/package/@polkadot/api-derive) methods:
```

Comment (ltfschoen) [inline packages/types/README.md]: `are are` should be `are`

Comment (Stefie) [reply packages/types/README.md]: copied over to new, combined PR

**Comment by Stefie** [comment]
Closed in favour of https://github.com/polkadot-js/api/pull/898

**Pull request closed without merging**